### ISSUE TITLE
CANCEL_USE:SHIELD<ticks> Does not work properly

### ISSUE THREAD

**Issue by MrSal202** [GC-spigot/AdvancedEnchantments#3699]
### Describe the bug

As you'll see in the video, the shield does not de-activate like it would when an axe would hit it or like when an enderpearl enters cooldown.
Instead, once the player stops shielding, then they cannot shield again (and it looks a bit buggy, but that's not important).
If the player continues to hold down shield when CANCEL_USE has been called, they continue to shield.

### How to reproduce

Here is an example config:

deactivateshieldonblock:
  display: '%group-color%Shield Deactivate Enchant'
  description: |-
    Deactivate shield after blocking an attack.
  applies-to: Shields
  type: SHIELD_BLOCK
  group: COMMON
  applies:
    - SHIELD
  levels:
    '1':
      effects:
        - CANCEL_USE:SHIELD:100 @Victim

Let an enemy hit your shield and continue blocking- you will still be able to block.
Unblock after an enemy hits you, then try blocking- the shield will be correctly deactivated (but try to block over and over client-side).

### Screenshots / Videos

https://streamable.com/mu3dgx

### "/ae plinfo" link

https://paste.md-5.net/miyepupili

**Comment by Awpmeister2170** [comment]
Hello, make sure your target is Capitalised ( @Victim ) and see if this works for you

**Comment by MrSal202** [comment]
> Hello, make sure your target is Capitalised ( @victim ) and see if this works for you

I fixed my targeter formatting to '@Victim' yet the issue persists.

**Comment by MrSal202** [comment]
> > Hello, make sure your target is Capitalised ( @victim ) and see if this works for you
> 
> I fixed my targeter formatting to '@victim' yet the issue persists.

Sorry, I meant '@Victim'

**Comment by MrSal202** [comment]
> > > Hello, make sure your target is Capitalised ( @victim ) and see if this works for you
> > 
> > 
> > I fixed my targeter formatting to '@victim' yet the issue persists.
> 
> Sorry, I meant '@victim'

It is capitalized, I dont know why its not capitalizing in my comment. I'm not familiar with github.

**Comment by MrSal202** [comment]
It is capitalized, in Github, I guess putting an @ symbol before a word lowercases it.

**Comment by ThomasWega** [comment]
Would need to be done with packets. Might add this sometime in future, but currently not a priority.
